ui/src/message_popup/author/questions_page.jsx

```diff
@@ -59,18 +66,37 @@ export default React.createClass({
     return questionString.split(text).length;
   },
   
+  onQuestionsReceived(err, response){
+    const allDatabaseQuestions = JSON.parse(response.text).row;
```

Comment:
This is related to what we were talking about in https://github.com/mit-teaching-systems-lab/threeflows/pull/114/files#diff-0861d6d6b50d7d695344bf2d86d6e5e6R194, although perhaps I missed it earlier.  Seeing `row` here feels like it's a detail of how we're storing things in the database, and doesn't help communicate what this data actually is.

Based on the code here, it looks like the endpoint will return `allDatabaseQuestions` which is an object like `{ currentQuestions: [], archivedQuestions: [] }`.  So you could remove the `row` wrapped in the server and in the parsing here, or you could change the `row` wrapper to `allDatabaseQuestions` to communicate what it is.